lang/golang/golang/Makefile

```diff
@@ -95,6 +95,12 @@ BOOTSTRAP_1_17_HASH:=a1a48b23afb206f95e7bbaa9b898d965f90826f6f1d1fc0c1d784ada0cd
 
 BOOTSTRAP_1_17_BUILD_DIR:=$(HOST_BUILD_DIR)/.go_bootstrap_1.17
 
+BOOTSTRAP_1_21_SOURCE:=go1.21.5.src.tar.gz
```

Comment:
basically we would use the oldest go toolchain for bootstrap, i.e. go1.20.15.

Reply:
Since this is already verified on OpenWrt and does not require any patches on our side (iirc 1.20 needs some patches), this should be good.

Reply:
> iirc 1.20 needs some patches

Not really, we used that version in the past.